internal/scorecard/alpha/formatting.go

```diff
@@ -24,7 +24,7 @@ import (
 
 // getTestResult fetches the test pod log and converts it into
 // ScorecardOutput format
-func (o Scorecard) getTestResult(ctx context.Context, p *v1.Pod, test Test) (output v1alpha2.ScorecardTestResult) {
+func (o Scorecard) getTestResult(ctx context.Context, p *v1.Pod, test Test) (output *v1alpha2.ScorecardTestResult) {
```

Comment:
Should this be a method on `PodTestRunner`?

Reply:
made it a method on PodTestRunner.